lib/mix/tasks/docs.ex

```diff
@@ -10,11 +10,11 @@ defmodule Mix.Tasks.Docs do
   ## Command line options
 
     * `--canonical`, `-n` - Indicate the preferred URL with
-      rel="canonical" link element, defaults to no canonical path
+      `rel="canonical"` link element, defaults to no canonical path
 
-    * `--formatter`, `-f` - Which formatters to use, "html" or
-      "epub". This option can be given more than once. By default,
-      both html and epub are generated.
+    * `--formatter`, `-f` - Which formatters to use, `"html"` or
+      `"epub"`. This option can be given more than once. By default,
+      both `html` and `epub` are generated.
```

Comment:
since these are CLI flags I think we can drop string quotes.

```suggestion
    * `--formatter`, `-f` - Which formatters to use, `html` or
      `epub`. This option can be given more than once. By default,
      both `html` and `epub` are generated.
```